qa/tasks/cephfs/test_misc.py

```diff
@@ -504,6 +504,27 @@ def test_dump_inmemory_log_on_missed_internal_heartbeats(self):
                 return
         self.assertTrue(False, "Failed to dump in-memory logs during missed internal heartbeat")
 
+    def _session_client_evict(self, cmd):
+        mount_a_client_id = self.mount_a.get_global_id()
+        info = self.fs.rank_asok(cmd + ['ls'])
+        self.assertIsNotNone(info)
+        size = len(info)
+        with self.assertRaises(CommandFailedError) as ce:
+            info = self.fs.rank_asok(cmd + ['evict'])
```

Comment:
Did you mean to assert for something on `info` here?  If not, this assignment is redundant.

Reply:
Removed `info`, its not required.